presidio-analyzer/presidio_analyzer/__init__.py

```diff
@@ -14,10 +13,23 @@
 from presidio_analyzer.analyzer_engine import AnalyzerEngine
 from presidio_analyzer.analyzer_request import AnalyzerRequest
 
+# Define default loggers behavior
+
+# 1. presidio_analyzer logger
+
 logging.getLogger("presidio-analyzer").addHandler(logging.NullHandler())
 
+# 2. decision_process logger.
+# Setting the decision process trace here as we would want it
+# to be activated using a parameter to AnalyzeEngine and not by default.
+
+decision_process_logger = logging.getLogger("decision_process")
+ch = logging.StreamHandler()
+formatter = logging.Formatter("[%(asctime)s][%(name)s][%(levelname)s]%(message)s")
```

Comment:
Can this be taken from the toml definition?

Reply:
Discussed offline, no quick and easy solution.